ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/MediaPlayerService.kt

```diff
@@ -136,6 +148,99 @@ class MediaPlayerService : Service() {
         }
     }
 
+    override fun onGetRoot(
```

Comment:
This is the key part of the browsing interface. Usage is spelled out [in this doc](https://developer.android.com/training/cars/media#onGetRoot). Note that the root itself isn't actually displayed in Android Auto.